client/src/app.tsx

```diff
@@ -143,7 +144,9 @@ const token = localStorage.getItem(process.env.AUTH_TOKEN);
 ReactDOM.render(
   <Provider store={store}>
     <ApolloProvider client={client as any}>
-      <Routes />
+      <ApolloHooksProvider client={client as any}>
```

Comment:
why do you need a `as any` here ?

Reply:
Hummm, that's a nice pick. I previously use `client as any` for `ApolloProvider` because of a mismatch in types between `ApolloProvider` and `ApolloClient` and simply assumed it would be the same for `ApolloHooksProvider`.

Actually, everything seems to be in order now, and I removed both "uncasts" in 5e6b787